index.html

```diff
@@ -4355,52 +4355,62 @@ <h1>Artists</h1>
+                            <img id="ambrosiaLuck1" src="Pictures/Default/BlueberryLuck.png">
+                        </div>
                     </div>
-                    <div class="blueberryUpgrade">
-                        <img id="ambrosiaLuck2" src="Pictures/Default/BlueberryLuck.png">
+                    <div class="blueberryUpgradeTier">
+                        <div class="blueberryUpgrade">
+                            <img id="ambrosiaCubeQuark1" src="Pictures/Default/BlueberryCubeQuark.png">
+                        </div>
+                        <div class="blueberryUpgrade">
+                            <img id="ambrosiaLuckQuark1" src="Pictures/Default/BlueberryLuckQuark.png">
+                        </div>
+                        <div class="blueberryUpgrade">
+                            <img id="ambrosiaLuckCube1" src="Pictures/Default/BlueberryLuckCube.png">
+                        </div>
+                        <div class="blueberryUpgrade">
+                            <img id="ambrosiaQuarkCube1" src="Pictures/Default/BlueberryQuarkCube.png">
+                        </div>
+                        <div class="blueberryUpgrade">
+                            <img id="ambrosiaCubeLuck1" src="Pictures/Default/BlueberryCubeLuck.png">
+                        </div>
+                        <div class="blueberryUpgrade">
+                            <img id="ambrosiaQuarkLuck1" src="Pictures/Default/BlueberryQuarkLuck.png">
+                        </div>
                     </div>
-                    <div class="blueberryUpgrade">
-                        <img id="ambrosiaPatreon" src="Pictures/Default/BlueberryPatreon.png">
+                    <div class="blueberryUpgradeTier">
+                        <div class="blueberryUpgrade">
+                            <img id="ambrosiaQuarks2" src="Pictures/Default/BlueberryQuarks.png">
+                        </div>
+                        <div class="blueberryUpgrade">
+                            <img id="ambrosiaCubes2" src="Pictures/Default/BlueberryCubes.png">
+                        </div>
+                        <div class="blueberryUpgrade">
+                            <img id="ambrosiaLuck2" src="Pictures/Default/BlueberryLuck.png">
+                        </div>
                     </div>
+                    <div class="blueberryUpgradeTier">
                         <button id="refundBlueberries" style="border: 2px solid blue" i18n="ambrosia.refundWord"></button>
                         <button id="getBlueberries" style="border: 2px solid blue" i18n="ambrosia.exportTree.exportButton"></button>
                         <label id="importBlueberriesButton" label="importBlueberriesButton" for="importBlueberries" style="border: 2px solid blue" i18n="settings.loadFromFile"></label>
                         <input type="file" id="importBlueberries" label="importBlueberries" accept=".txt" style="display: none">
+                    </div>
```

Comment:
They're obviously not quite a _tier_, but reusing the class here as well ensures that the buttons all stay in the same row; that's the part that matters.